src/Symfony/Component/DependencyInjection/Tests/Fixtures/config/services_closure_argument.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+return function (ContainerConfigurator $c) {
+    $s = $c->services()->defaults()->public();
+
+    $s->set('foo', 'Foo');
+
+    $s->set('service_closure', 'Bar')
+        ->args([service_closure(service('foo'))]);
+
+    $s->set('service_closure_invalid', 'Bar')
+        ->args([service_closure(service('invalid_service')->nullOnInvalid())]);
```

Comment:
it'd be great to be able to do this instead:
`service_closure('invalid_service')->nullOnInvalid()`

maybe with a new `ClosureReferenceConfigurator`?

Reply:
@nicolas-grekas Done. The fabbot failure is unrelated to my changes, I can apply it if you want.